src/Apps/Artwork/Components/PrivateArtwork/PrivateArtworkAboutArtist.tsx

```diff
@@ -32,7 +32,7 @@ export const PrivateArtworkAboutArtist: React.FC<React.PropsWithChildren<
       fragment PrivateArtworkAboutArtist_artwork on Artwork {
         displayArtistBio
         slug
-        artists(shallow: true) {
+        privateArtists: artists(shallow: false) {
```

Comment:
This is a relay thing. 

Had to alias because everywhere else in this query tree we're doing `artists(shallow: true)`, and so when one field has a difference, it creates a resolution conflict when things are rolled up and sent to the graphql server. Aliases fix it though.